runtime/help/colors.md

```diff
@@ -89,13 +94,13 @@ environment variable `COLORTERM` should have the value `truecolor`, `24bit`,
 or `24-bit`. In addition, to enable true color in micro, the environment
 variable `MICRO_TRUECOLOR` must be set to 1.
 
-* `solarized-tc`: this is the solarized colorscheme for true color.
-* `atom-dark-tc`: this colorscheme is based off of Atom's "dark" colorscheme.
-* `cmc-tc`: A true colour variant of the cmc theme.  It requires true color to
-  look its best. Use cmc-16 if your terminal doesn't support true color.
-* `gruvbox-tc`: The true color version of the gruvbox colorscheme
-* `github-tc`: The true color version of the Github colorscheme
```

Comment:
noticed this is now gone, will a newer version to reflect new dark mode be made available?